File: examples/src/Slides/Transitions/SetTransitionMorphType .cs
import aspose.slides as slides
using Aspose.slides.Examples.CSharp
using Aspose.slides.Export
using Aspose.slides.SlideShow
using System
using System.Collections.Generic
using System.Linq
using System.text
using System.Threading.Tasks

namespace CSharp.slides.Transitions
{
    class SetTransitionMorphType
    {
        public static void Run() {

            #ExStart:SetTransitionMorphType
            # The path to the documents directory.
            dataDir = RunExamples.GetDataDir_Slides_Presentations_Transitions()

            using (Presentation presentation = new Presentation(dataDir + "presentation.pptx"))
            {
                presentation.slides[0].SlideShowTransition.type = TransitionType.Morph
                ((IMorphTransition)presentation.slides[0].SlideShowTransition.value).MorphType = TransitionMorphType.ByWord
                presentation.save(dataDir + "presentation-out.pptx", slides.export.SaveFormat.PPTX)
            }
            #ExEnd:SetTransitionMorphType
        }
    }
}

File: examples/src/Slides/Transitions/set_transition_morph_type.py
import aspose.slides as slides


def set_transition_morph_type(global_opts):
    with slides.Presentation(global_opts.data_dir + "welcome-to-powerpoint.pptx") as presentation:
        presentation.slides[0].slide_show_transition.type = slides.slideshow.TransitionType.MORPH
        presentation.slides[0].slide_show_transition.value.morph_type = slides.slideshow.TransitionMorphType.BY_WORD
        presentation.save(global_opts.out_dir + "transition_MORPH_out.pptx", slides.export.SaveFormat.PPTX)
